sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -472,6 +472,27 @@ def test_KNeighborsRegressor_multioutput_uniform_weight():
         assert_array_almost_equal(y_pred, y_pred_idx)
 
 
+def test_radius_neighbors_regressor_zero_distance():
```

Comment:
I'm not sure what other test you were referring, but I've added the `KNeighborsRegressor